packages/strapi-admin/admin/src/containers/InstallPluginPage/styles.scss

```diff
@@ -1,8 +1,7 @@
 .containerFluid {
   padding: 18px 30px !important;
   > div:first-child {
-    max-height: 33px;
-    margin-bottom: 48px;
+    margin-bottom: 12px;
```

Comment:
To keep the alignement `11px` is better
![screenshot 2018-11-13 at 09 57 25](https://user-images.githubusercontent.com/13311463/48401913-8ae97b00-e72a-11e8-9d55-e51462ea2a1c.png)

Reply:
I will change it in the day 👍